source/fe/mapping_q_generic.cc

```diff
@@ -3810,11 +3810,17 @@ add_line_support_points (const typename Triangulation<dim,spacedim>::cell_iterat
             }
           else
             {
-              tmp_points.resize(2);
-              tmp_points[0] = cell->vertex(GeometryInfo<dim>::line_to_cell_vertices(line_no, 0));
-              tmp_points[1] = cell->vertex(GeometryInfo<dim>::line_to_cell_vertices(line_no, 1));
-              manifold.add_new_points(tmp_points,
-                                      support_point_weights_perimeter_to_interior[0], a);
+              std::array<Point<spacedim>, 2> vertices;
+              vertices[0] = cell->vertex(GeometryInfo<dim>::line_to_cell_vertices(line_no, 0));
+              vertices[1] = cell->vertex(GeometryInfo<dim>::line_to_cell_vertices(line_no, 1));
```

Comment:
Could you make the variable `const` by brace-initializing the pair? (Not sure if that works for `std::array`, though.)

Reply:
Had to try it out -- it works.

Reply:
I really loathe the way astyle indents the brace initializers but we may as well do it. Fixed.